### DISCUSSION TITLE
不用进入视频播放就能  快速收藏

### DISCUSSION THREAD

**Discussion by ChitaGideon** [the1812/Bilibili-Evolved#4761] [功能建议 / Ideas]
<img alt="Shows an illustrated sun in light mode and a moon with stars in dark mode." src="https://private-user-images.githubusercontent.com/86563828/297708297-1df1a484-ed7b-48f9-b1ff-9569969b82bd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY2MDc4NjgsIm5iZiI6MTcxNjYwNzU2OCwicGF0aCI6Ii84NjU2MzgyOC8yOTc3MDgyOTctMWRmMWE0ODQtZWQ3Yi00OGY5LWIxZmYtOTU2OTk2OWI4MmJkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI1VDAzMjYwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFmMmFiZjJmZGI1OGU1ZjY0MmI1NmM4Yjk3ODhjZjU1YTdkOThmNDE4MzA2N2JlYWU2OGYzNDQ2NjUyYWE1YTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.24qWKWeLy11a_sX8Bski5XVGtUWUfs2utWaxIXbaAo8">

比如就在 稍后再看按钮  的旁边加个 快速收藏的按钮， 用起来方便

**Comment by the1812** [comment]
都没点开看为什么要收藏

**Reply by ChitaGideon** [reply]
看之前提前分类收藏，有些长视频不方便近期看呢。特别是推荐页的，不收藏再找就找不到了

**Reply by the1812** [reply]
那不就是应该放稍后再看里？

**Reply by ChitaGideon** [reply]
那后面需要再分一次类啊；

比如我去搜索英语公开课，需要直接把感兴趣的视频都收藏到 英语学习 收藏夹下；不用每次都点进去再收藏

**Reply by Geeyun-JY3** [reply]
把破站的收藏夹等同为 YouTube 的播放列表这个需求是不是就好理解些了（当然 YouTube 也没有本讨论所提到的相关功能）。

**Reply by yjcaaaaaaaa** [reply]
复议 还有可能嘛这个功能?(^ ^)
